packages/features/bookings/Booker/components/Header.tsx

```diff
@@ -32,7 +32,7 @@ export function Header({
     [setLayout]
   );
 
-  if (isMobile || !enabledLayouts || enabledLayouts.length <= 1) return null;
```

Comment:
We can't return early since we need to show the header for the column and week view. That bar contains the title and prev/next week buttons. That's why the check is moved further down.